tests/test_cache_lock_file_perf.py

```diff
@@ -0,0 +1,110 @@
+import multiprocessing
+import threading
+import time
+from msal_extensions import FilePersistence, CrossPlatLock
+
+
+def validate_thread_locking_file(num):
```

Comment:
`num` is a pretty common noun in programming. Consider using longer, more descriptive names for variables. `expectedThreadCount` in this case would make this method more readable.